### ISSUE TITLE
Support for Drush 8 and Drush 9 Needed in BLT 9

### ISSUE THREAD

**Issue by aweingarten** [acquia/blt#2729]
As a BLT user of the BLT 9.x branch I need support for both Drush 8 and Drush 9.

This allows me to use legacy drush commands from themes like COG, and modules like AcquiaPurge which have not yet updated their commands,

I also don't want to have to worry about upgrading all my associated modules at the same time as drush. The transition from drush 8 to drush 9 shold be a gradual move.

Also see
https://github.com/acquia-pso/cog/issues/145

TLDR: 
There is no current solution for themes to expose commands.  
https://github.com/drush-ops/drush/pull/3089
https://www.drupal.org/project/drupal/issues/2002606#comment-10786044

**Comment by malikkotob** [comment]
I'm not sure how BLT should address this. [The official drush documentation](http://docs.drush.org/en/master/install/#drupal-compatibility) notes that drush 8 technically only supports Drupal core versions 8.3-, where drush 9 is technically designated to support core 8.4+. It seems like porting themes/modules to use 9.x flavors of drush commands would be the right path forward. [Here's a long thread](https://www.drupal.org/project/drupal/issues/2906637) where drush maintainers indicate that although drush 8 currently works with core 8.4+, it's not supported.

**Comment by grasmash** [comment]
Closing as duplicate of https://github.com/acquia/blt/issues/2731
